apps/api/src/domain/models/term/entities/term.from-event-history.spec.ts

```diff
@@ -154,6 +155,25 @@ describe(`Term.fromEventHistory`, () => {
                 });
             });
 
+            describe(`when a term has been deleted`, () => {
```

Comment:
I wonder if we can do this generically with a `Widget` instead of a term. We don't really need to test `handleResourceDeleted` everywhere. Also, if we side-step the issue by filtering events for deleted aggregates in the repo, this becomes a non-issue.